### PR TITLE
Make export lists admin configurable

### PR DESCRIPTION
**Pull request by alexabird**
**To test**

1. Login as admin
2. Upload some products
3. Run search
4. Attempt to export single item
5. See that the full export list is available
6. Attempt to export multiple items
7. See that the full export list is available
8. Go to admin ui > catalog ui search
9. Configure the export format lists (there should be a description listing the valid formats)
10. attempt exports again and see the format options updated

### CODE REVIEW

**Review by pvargas — APPROVED** [review]

File: ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml [hunk 1]
```diff
@@ -27,6 +27,19 @@
             type="Integer"
             default="1000"/>
 
+
+        <AD id="exportMetacardFormatOptions"
+            name="Export Metacard Format Options"
+            description="Specifies the export format option available for export of a single metacard. This should be a comma separated list. Valis options are: audio, csv, csw:Record, deployed-harvested-resource, geojson, gmd:MD_Metadata, html, kml, kmz, resource, rtf, thumbnail, txt, xlsx, xml."
```
Comment (jrnorth) [inline ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml]: format option -> format options
Valis -> Valid

File: ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml [hunk 2]
```diff
@@ -27,6 +27,19 @@
...
+            type="String"
+            default="audio,csv,csw:Record,deployed-harvested-resource,geojson,gmd:MD_Metadata,html,kml,kmz,resource,rtf,thumbnail,txt,xlsx,xml"/>
+
+        <AD id="exportMetacardsFormatOptions"
+            name="Export Multiple Metacard Format Options"
+            description="Specifies the export format option available for export of a single metacard. This should be a comma separated list. Valid options are: atom, csv, csw:Record, geojson, kml, kmz, xml, html, rtf, xlsx."
```
Comment (jrnorth) [inline ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml]: a single metacard -> multiple metacards
format option -> format options

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/export/export.tsx [hunk 3]
```diff
@@ -77,7 +82,44 @@ export const getExportResult = (result: LazyQueryResult) => {
...
+        const newFormats = configuredFormats
+          .map((configuredFormat: string) => {
+            const validFormat = exportFormats.find(
+              (exportFormat: ExportFormat) =>
+                exportFormat.id === configuredFormat
+            )
+            if (validFormat == undefined)
+              console.log(
+                configuredFormat +
+                  ' does not match any valid transformers; Cannot not include format in export list.'
```
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/export/export.tsx]: ✏️ cannot include format in export list.

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
build now

**Pull request merged**